graphql/tracing/prometheus.go

```diff
@@ -0,0 +1,137 @@
+package tracing
+
+import (
+	"context"
+
+	time "github.com/echlebek/timeproxy"
+	"github.com/graphql-go/graphql"
+	"github.com/graphql-go/graphql/gqlerrors"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/sensu/sensu-go/util/strings"
```

Comment:
Might want to use an alias here, as not to get confused with `strings` from the standard library.

```suggestion
	utilstrings "github.com/sensu/sensu-go/util/strings"
```

Reply:
addressed 🙏